fbw-a32nx/src/systems/instruments/src/EFB/index.tsx

```diff
@@ -4,8 +4,18 @@
 import React from 'react';
 
 import { render } from '@instruments/common/index';
-import { EfbWrapper } from '@flybywiresim/flypad';
+import { EfbWrapper, PerformanceCalculatorContext } from '@flybywiresim/flypad';
 import { A320FailureDefinitions } from '@failures';
+import { A320251NLandingCalculator } from '@shared/performance/a32nx_landing';
```

Comment:
This namespacing is really ugly but too bad, it will have to be like this for now.

Reply:
Open to suggestions. It is a landing calculator for that exact type.

Reply:
Root of the problem is, ideally we'd want the EFB be generic for all FBW a/c types, so we would just have a generic landing calculator FC LandingCalculator that doesn't need awareness of what the variant the plane is, and some input params to that that would be defined per variant (maybe using the VFS config OR xmlConfig (Note: I have no idea how to import XML config into react (probably not possible as we do not inherit BaseInstruments in react displays). 

Most likely we'd have to wait until after flyPad OS 4 (4.1? 5?) to properly abstract this away.

Reply:
That generic interface is already implemented here. This is the aircraft-specific context that provides the calculator (through the generic interface) to the generic EFB.

Reply:
This is literally the point of commit 2ccf15c3a9b387b91b4183e8b23bcfb6f8d0ac6e.